index.html

```diff
@@ -1,14 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello Microverse</title>
+    <title>Temesgen Portfolio</title>
     <link rel="stylesheet" href="style.css">
 
 </head>
+
 <body>
-    <h1 class="main-header">Hello Microverse!</h1>
+    <header>
+        <h4>My Logo</h4>
+        <nav>
+            <a href="">
+            </a>
+            <img src="./img/menu.png" alt="web menu graphics">
+        </nav>
+    </header>
+
+    <main>
+        <section class="Profile">
+                <h1>I’m Temesgen <br> Glad to see you!</h1>
```

Comment:
As required by the previous reviewer, please update the design to stick to Figma.

Please update the size and the color 
Your design:
<img width="318" alt="image" src="https://user-images.githubusercontent.com/7243132/170825527-d1fe5b3c-c98f-496a-8921-ecf4e658ef83.png">

Figma:
<img width="442" alt="image" src="https://user-images.githubusercontent.com/7243132/170825533-ae54f32a-8f36-40c9-a346-85a15a3e1e52.png">